resume/resume-2.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Nick Long Resume</title>
+	</head>
+	<body>
+		<style>
+	body {
+		background-color:rgb(243, 240, 234);
+		font-family: monospace;
+	}
+	#contact {
+		text-align: center;
+	}
+	h2 {
+		font-style: italic;
+	}
+	.collapsible {
+		cursor: pointer;
```

Comment:
adds a little hover bit when you mouse over to make it look like you can click it: https://www.w3schools.com/cssref/playit.asp?filename=playcss_cursor&preval=pointer

Reply:
THAT'S FUCKING PERFECT! I was looking for a way let people know they could click it.